shared/src/commonTest/kotlin/org/hyperskill/DateFormatterTest.kt

```diff
@@ -1,91 +1,101 @@
 package org.hyperskill
 
-import kotlin.test.Test
-import kotlin.test.assertEquals
-import org.hyperskill.app.core.view.mapper.DateFormatter
+import kotlin.time.DurationUnit
+import kotlin.time.toDuration
+import org.hyperskill.app.core.view.mapper.SharedDateFormatter
 
 class DateFormatterTest {
-    @Test
-    fun testMinutes() {
-        assertEquals(
-            DateFormatter.formatTimeDistance(3 * DateFormatter.ONE_MINUTE_IN_MILLIS + 17 * 1000),
-            "3 minutes"
-        )
-
-        assertEquals(
-            DateFormatter.formatTimeDistance(3 * DateFormatter.ONE_MINUTE_IN_MILLIS + 37 * 1000),
-            "4 minutes"
-        )
-
-        assertEquals(
-            DateFormatter.formatTimeDistance(
-                17 * DateFormatter.ONE_MINUTE_IN_MILLIS + DateFormatter.THIRTY_SEC_IN_MILLIS
-            ),
-            "18 minutes"
-        )
-    }
-
-    @Test
-    fun testHours() {
-        assertEquals(
-            DateFormatter.formatTimeDistance(49 * DateFormatter.ONE_MINUTE_IN_MILLIS),
-            "about 1 hour"
-        )
-
-        assertEquals(
-            DateFormatter.formatTimeDistance(
-                7 * DateFormatter.ONE_HOUR_IN_MILLIS + 17 * DateFormatter.ONE_MINUTE_IN_MILLIS
-            ),
-            "about 7 hours"
-        )
-
-        assertEquals(
-            DateFormatter.formatTimeDistance(
-                23 * DateFormatter.ONE_HOUR_IN_MILLIS +
-                    59 * DateFormatter.ONE_MINUTE_IN_MILLIS + DateFormatter.THIRTY_SEC_IN_MILLIS
-            ),
-            "about 24 hours"
-        )
+    companion object {
+        val ONE_SECOND_IN_MILLIS: Long = 1.toDuration(DurationUnit.SECONDS).inWholeMilliseconds
+        val ONE_MINUTE_IN_MILLIS: Long = 1.toDuration(DurationUnit.MINUTES).inWholeMilliseconds
+        val ONE_HOUR_IN_MILLIS: Long = 1.toDuration(DurationUnit.HOURS).inWholeMilliseconds
     }
 
-    @Test
-    fun testMonths() {
-        assertEquals(
-            DateFormatter.formatTimeDistance(DateFormatter.ONE_MINUTE_IN_MILLIS * 60 * 24 * 41),
-            "about 1 month"
-        )
+    private val dateFormatter = SharedDateFormatter(ResourceProviderStub())
 
-        assertEquals(
-            DateFormatter.formatTimeDistance(DateFormatter.ONE_MINUTE_IN_MILLIS * 60 * 24 * 47),
-            "about 2 months"
-        )
+    // TODO: Implement tests
```

Comment:
Please, add the reason why tests are commented to the TODO.

Reply:
@XanderZhu JFYI https://vyahhi.myjetbrains.com/youtrack/issue/ALTAPPS-489/Shared-Unified-shared-Formatter-for-date-and-time#focus=Comments-74-233329.0-0

Reply:
Task about adding tests https://vyahhi.myjetbrains.com/youtrack/issue/ALTAPPS-784

Reply:
@ivan-magda 
Yes, I know the reason. I think tests should not be just commented. It's better to describe the reason why they are commented in place for future readers.